brands/shop/newsagent.json

```
@@ -41,6 +41,36 @@
       "shop": "newsagent"
     }
   },
+  "shop/newsagent|Martin's": {
+    "count": 53,
+    "countryCodes": ["gb"],
+    "match": [
+      "shop/convenience|Martin's",
+      "shop/newsagent|McColls"
+    ],
+    "tags": {
+      "brand": "Martin's",
+      "brand:wikidata": "Q7277785",
+      "brand:wikipedia": "en:RS McColl",
+      "name": "Martin's",
+      "shop": "newsagent"
+    }
+  },
+  "shop/newsagent|McColls": {
+    "count": 53,
```

This entry did not exist before too, so it should also be
```suggestion
    "nocount": true,
```